src/dev/run_check_file_casing.ts

```diff
@@ -26,6 +26,9 @@ run(async ({ log }) => {
       // so it's still super slow. This prevents loading the files
       // and still relies on gitignore to final ignores
       '**/node_modules',
+      // Ignore the dashboard files introduced in newer version when the 
```

Comment:
This is needed because in newer versions, when we introduced the dashboard names, the rule was ignored ([here](https://github.com/elastic/kibana/blob/4b7f8d9399503ce0c5c93e8ee844398dd2199810/src/dev/run_check_file_casing.ts#L29-L36)). This allowed us to use the name pattern not matching the standard `snake_case`. To skip this check, I added only the folder with the dashboards to the list, just to make the backporting possible, but in the long run, we should rename those to match the pattern ( I will add an issue for that )

Reply:
Issue added https://github.com/elastic/kibana/issues/222795